lib/screens/wrapper.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/material.dart';
+import 'package:innopolis_feedback/main.dart';
+import 'package:innopolis_feedback/screens/authenticate/authenticate.dart';
+import 'package:provider/provider.dart';
+
+class Wrapper extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
```

Comment:
Extra file and code. This boolean check could be made in main file.

Reply:
how?